tests/foreman/api/test_activationkey.py

```diff
@@ -307,11 +296,6 @@ def test_positive_add_host_collections(self):
         act_key = act_key.update(['host_collection'])
         self.assertEqual(len(act_key.host_collection), 2)
 
-        # Give activation key zero host collections.
```

Comment:
Removing that verification as direct PUT call to entity should not disassociate host collections from activation key. We have another path to perform that particular operation.
+ That verification is definitely is out of scope `test_positive_add*` method.
Needs to add coverage in that area in future